app/styles/ui/_pull-request-badge.scss

```diff
@@ -0,0 +1,28 @@
+.toolbar-dropdown.open .pr-badge {
+  background: $gray-200;
```

Comment:
I realize that I forgot to spec the open button. Could we please bump this to 300?